framework/widgets/Pjax.php

```diff
@@ -55,12 +55,20 @@ class Pjax extends Widget
      */
     public $linkSelector;
     /**
+     * @var boolean whether to enable auto generated link selector.
```

Comment:
It's not generating just a selector. It's pjax handler for all links matching selector.